components/data/Socials.js

```diff
@@ -7,36 +7,37 @@ import {
 	FaGithub,
 } from "react-icons/fa";
 
+const classNames = "text-[5vw]";
```

Comment:
put this directly into the className, it does get repetitive but cleaner to understand cause everything you need is within that line

Reply:
oh okay